_playwright-tests/tests/API/PackageGroups.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { expectError, test } from './base_client';
+import {
+  ApiRepositoryResponse,
+  GetRepositoryRequest,
+  type ListRepositoriesRequest,
+  PackagegroupsApi,
+  RepositoriesApi,
+} from './client';
+import { expect } from '@playwright/test';
+import { randomName } from './helpers/repoHelpers';
+import { poll } from './helpers/apiHelpers';
+import { randomUUID } from 'crypto';
+
+test.describe('Package groups', () => {
+  test('Search and list package groups', async ({ client }) => {
+    const repoUrl = 'https://content-services.github.io/fixtures/yum/comps-modules/v1/';
+    const repoName = randomName();
+    let repoUuid: string | undefined;
```

Comment:
nit: this could be declared just as a string and then expected and asserted as non-null right after repo creation, this would eliminate the need to use the bang (non-null assertion) several times after that 😅